app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="container-fluid p-0" style="max-width: 100%;">
+  <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #3778c2;">
```

Comment:
- [ ] Kindly style your Navbars for the Registration and Login pages as seen in the Given design :+1:

| Your Design | The Given Design |
|--------|--------|
| ![image](https://github.com/RileyManda/coincraft/assets/102699958/51346405-7655-4d83-b00e-28f3024d4a62)| ![image](https://github.com/RileyManda/coincraft/assets/102699958/9267fb06-afc4-4763-95a0-b50db3039719) |